app/models/test.rb

```diff
@@ -1,7 +1,7 @@
 class Test < ApplicationRecord
   belongs_to :category
 
-  def self.sort_reverse(category)
-    where(category_id: Category.find_by(title: category).id).order('title DESC').pluck(:title)
+  def self.sort_reverse category
```

Comment:
```suggestion
  def self.sort_reverse(category)
```
без скобок не рекомендуют писать
тем более дефиниции
при вызове можно без скобок, но обычно это относится к каким-то встроенным методам руби/рельс
или методам кастомным, но специфическим, которые добавляют какое-то поведение классу глобально... т.е. методам уровня `belongs_to`
(во 2-ой строке  - это как вызов метода без скобок)